static/styles/popovers.css

```diff
@@ -355,6 +355,12 @@ ul {
             &[data-type="2"] .value {
                 overflow-wrap: break-word;
             }
+
+            &[data-type="5"] {
+                overflow: hidden;
+                text-overflow: ellipsis;
+                max-width: 300px;
```

Comment:
How did you pick 300px?  I think we probably want this to just be the max width of the panel.

Please also add a comment above this block noting what type=5 translates to; and the same for the "2" above.

Also, the commit message is not helpful; it just repeats the code. You should instead explain __why__ this is correct, e.g. that'd be a good place to say why 300px is the right number.